### PR TITLE
[Feature] Implement main page

### LINKED ISSUE #26: [Feature] Group Search in MainPage
## Todos
 - [ ] Implement Group Search in MainPage

## Related Issues
- #17

### PR DESCRIPTION
**Pull request by arclic**
### MainPage
스크린샷 2021-11-12 오후 6 15 31

스크린샷 2021-11-12 오후 6 15 38

- Design Group/Content tab
- Implement GroupSearch feature
- Implement Group/Content List Item components
- Modify backend to communicate well with frontend

### Tests

스크린샷 2021-11-12 오후 6 15 59

스크린샷 2021-11-12 오후 6 16 09

## Related Issues
- #17 
- close #26

### CODE REVIEW

File: backend/subroker/views.py [hunk 1]
```diff
@@ -88,15 +89,37 @@ def logout(request):
...
+            group_all_list = [{
+                'id': group.id,
+                'platform': group.membership.ott,
+                'membership': group.membership.membership,
+                'title': group.name,
+                'leader': group.leader.username,
+                'price': group.membership.cost,
+                'curMember': group.current_people,
+                'maxMember': group.membership.max_people,
+                'duration': group.payday
```
Comment (cw10108) [inline backend/subroker/views.py:120]: maybe payday would be appropriate

File: backend/subroker/views.py [hunk 2]
```diff
@@ -88,15 +89,37 @@ def logout(request):
...
+                    [ott, membership] = _query.split('__')
+                    Q_ott.add(Q(membership__ott__iexact=ott) & Q(membership__membership__iexact=membership), Q.OR)
+
+                groups = groups.filter(Q_ott)
+
+            # TODO
+            # fix each name
+            group_all_list = [{
+                'id': group.id,
+                'platform': group.membership.ott,
```
Comment (cw10108) [inline backend/subroker/views.py:113]: I think 'ott' would be better, for consistency

File: backend/subroker/views.py [hunk 3]
```diff
@@ -88,15 +89,37 @@ def logout(request):
...
+                'membership': group.membership.membership,
+                'title': group.name,
```
Comment (cw10108) [inline backend/subroker/views.py:115]: title is good, so we may change our model field to group.title

**Review by cw10108 — COMMENTED** [review]
we should discuss on our backend and frontend

File: backend/subroker/views.py (hunk 2 above)
Comment (arclic) [reply backend/subroker/views.py:113]: @cw10108 I think we should re-define our model and JSON type for front-end and back-end in next sprint

File: backend/subroker/views.py (hunk 3 above)
Comment (arclic) [reply backend/subroker/views.py:115]: @cw10108 same as above.

**Review by muhwagwa — APPROVED** [review]

**Pull request merged**